babeld/files/babeld.init

```diff
@@ -113,34 +113,39 @@ babel_config_cb() {
 		}
 	;;
 	"interface")
+	{
+		local _ignored
+		config_get_bool _ignored "$section" 'ignore' 0
+		[ "$_ignored" == "1" ] && return
```

Comment:
If you are moving this around anyways, maybe change to single equal sign? File has /bin/sh shebang and other comparisons in this file are also using single = ...

Reply:
Is this an intended behavior change:
Previously, option_cb() was set to do nothing if interface was ignored.
Now, option_cb() is not set at all, since you return here...

Reply:
so should I define `option_cb() { return; }` to make sure nothing is done?

Reply:
If I understand the original code correctly, it would be 
```
[ "$_ignored" == "1" ] && 
{
option_cb() { return; }
return
}
```
which is terribly ugly and may be the case why it was solved with the other order initially.

However, I not familiar with this script and just stumbled over this issue when reading the PR. So my initial comment was more a question than a statement.